site/content/en/docs/manual/advanced/annotation-quality.md

```diff
@@ -0,0 +1,193 @@
+---
+title: 'Annotation quality & Honeypot'
+linkTitle: 'Annotation quality'
+weight: 14
+description: 'How to check the quality of annotation in CVAT'
+---
+
+In CVAT, it's possible to evaluate the quality of annotation through
+the creation of a **Ground truth** job, referred to as a Honeypot.
+CVAT compares all other jobs in the specified tasks against the
+established **Ground truth** job and calculates annotation quality
+based on this comparison.
+
+> **Note** that Honeypot does not work with cubooids.
+
+See:
+
+- [Ground truth job](#ground-truth-job)
+- [Managing Ground Truth jobs: Import, Export, and Deletion](#managing-ground-truth-jobs-import-export-and-deletion)
+  - [Import](#import)
+  - [Export](#export)
+  - [Delete](#delete)
+- [Assessing data quality with Ground truth jobs](#assessing-data-quality-with-ground-truth-jobs)
+  - [Quality data](#quality-data)
+  - [Annotation quality settings](#annotation-quality-settings)
+  - [GT conflicts in the CVAT interface](#gt-conflicts-in-the-cvat-interface)
+- [Annotation quality \& Honeypot video tutorial](#annotation-quality--honeypot-video-tutorial)
+
+## Ground truth job
+
+You don’t need to annotate the whole dataset twice, the annotation quality of a small part of
+the data will show the quality of annotation for the whole dataset.
+
+You need to select several frames from the whole
+dataset depending on the size of the task.
+
+For the quality assurance to function properly, each job should contain at
+least a few overlapping frames. For example, for **task that contains only
+30 frames** to obtain adequate data, it's advisable to select **8-10 frames**.
+
+Depending on the dataset size, **5-15% of the data is good enough** for quality estimation.
+
+To create a **Ground truth** job, do the following:
+
+1. Create a [task](/docs/manual/basics/create_an_annotation_task/), and open the task page.
+2. Click **+**.
+
+   ![Create job](/images/honeypot01.jpg)
+
+3. In the **Add new job** window, fill in the following fields:
+
+   ![Add new job](/images/honeypot02.jpg)
+
+   - **Job type**: Use the default parameter **Ground truth**.
+   - **Frame selection method**: Use the default parameter **Random**.
+   - **Quantity %**: Set the desired percentage of frames for the "ground truth" job. <br>Note, that upon using **Quantity %**, the **Frames** field will be automatically populated.
+   - **Frame count**: Set the desired number of frames for the "ground truth" job. <br>Note, that upon using **Frames**, the **Quantity %** field will be automatically populated.
+
+   - **Seed**: (Optional) TBD
```

Comment:
```suggestion
   - **Seed**: (Optional) If you need to make the random selection reproducible, specify this number. It can be any integer number, the same value will yield the same random selection (given that the frame number is unchanged).
```